lib/tracing-limit/benches/limit.rs

```diff
@@ -102,18 +102,18 @@ impl<'a> field::Visit for Visitor<'a> {
 }
 
 impl tracing::Subscriber for VisitingSubscriber {
-    fn new_span(&self, span: &span::Attributes) -> Id {
+    fn new_span(&self, span: &span::Attributes<'_>) -> Id {
```

Comment:
What's the deal with these?

Reply:
The elided lifetimes are part of 2018 edition idioms.

Reply:
[Found it](https://doc.rust-lang.org/edition-guide/rust-2018/ownership-and-lifetimes/the-anonymous-lifetime.html#more-details), interesting